core/src/main/scala/io/projectglow/transformers/blockvariantsandsamples/VariantSampleBlockMaker.scala

```diff
@@ -27,6 +27,11 @@ import org.apache.spark.sql.types.{ArrayType, IntegerType, StringType}
 
 private[projectglow] object VariantSampleBlockMaker extends GlowLogging {
 
+  def filterOneDistinctValue(df: DataFrame): DataFrame = {
+    logger.warn("Filtered variants whose values were all the same.")
```

Comment:
nit: I'd saying "Filtering" to be clear that we didn't necessarily filter anything. This should probably also be an INFO instead of a WARN.